test/services/docker.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Docker Application Container Engine
+Documentation=http://docs.docker.com
+After=network.target docker.socket
+Requires=docker.socket
+
+[Service]
+ExecStart=/usr/bin/docker daemon -H fd:// --storage-driver=aufs
```

Comment:
it works for me if I `sudo modprobe aufs` before -  but [this](https://github.com/resin-io/resin-containers/pull/132/files) should make it unnecessary